src/server.rs

```diff
@@ -268,13 +271,14 @@ impl HttpServer {
     ///
     /// # Errors
     /// Returns an `IOError` when binding or `epoll::create` fails.
-    pub fn new<P: AsRef<Path>>(path_to_socket: P) -> Result<Self> {
+    pub fn new<P: AsRef<Path>>(path_to_socket: P, maybe_max_len: Option<usize>) -> Result<Self> {
         let socket = UnixListener::bind(path_to_socket).map_err(ServerError::IOError)?;
         let epoll = epoll::Epoll::new().map_err(ServerError::IOError)?;
         Ok(Self {
             socket,
             epoll,
             connections: HashMap::new(),
+            max_length: maybe_max_len.unwrap_or(MAX_PAYLOAD_SIZE),
```

Comment:
I feel this member of the struct needs a better name, "max_length" is not specific enough. Maybe "payload_max_size"?